code/datums/elements/food/fried_item.dm

```diff
@@ -0,0 +1,55 @@
+/// Items fried through the deep fryer.
+/datum/element/fried_item
+	/// List of colors to apply to fried food, each index is applied at a different level
+	var/static/list/fried_colors = list(
+		rgb(166, 103, 54),
```

Comment:
These should be color defines, it's impossible for me to know what these are on web without looking them up